contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -973,53 +973,52 @@ contract NodeOperatorsRegistry is AragonApp, IStakingModule, Versioned {
         return TYPE_POSITION.getStorageBytes32();
     }
 
-    /// @notice Returns the validators stats of all node operators in the staking module
-    /// @return exitedValidatorsCount Total number of validators in the EXITED state
-    /// @return activeValidatorsKeysCount Total number of validators in active state
-    /// @return readyToDepositValidatorsKeysCount Total number of validators ready to be deposited
-    function getValidatorsKeysStats()
-        external
-        view
-        returns (
-            uint256 exitedValidatorsCount,
-            uint256 activeValidatorsKeysCount,
-            uint256 readyToDepositValidatorsKeysCount
-        )
-    {
+    // struct ValidatorsReport {
+    //     uint256 totalExited;
+    //     uint256 totalDeposited;
+    //     uint256 totalVetted;
+    //     uint256 totalStuck;
+    //     uint256 totalRefunded;
+    //     uint256 targetLimit;
+    //     uint256 excessCount;
```

Comment:
what is `excessCount`? can it be calc from the other values?

Reply:
It's a new field introduced by #557 PR. According to the description: "excess validators count for operator that will be forced to exit"

Reply:
I thought it's done via `targetLimit`, prob I'm misunderstanding smth